src/System.IO.FileSystem/tests/FileStream/LockUnlock.cs

```diff
@@ -90,16 +86,16 @@ public void NonOverlappingRegions_Success(long fileLength, long firstPosition, l
             }
         }
 
-        [ActiveIssue(5964, TestPlatforms.AnyUnix)]
         [Theory]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
`Unix locks are on a per-process basis, so overlapping locks from the same process are allowed.`

How's that?